pages/support/[category]/index.js

```diff
@@ -41,83 +20,117 @@ const CategoryStyled = styled(Container).attrs({
   className: '',
 })``
 
-const Category = ({ supportServices, query: { category } }) => {
-  const [services, setServices] = useState(supportServices)
+const Category = ({ query: { category } }) => {
+  const [services, setServices] = useState([])
+  const [region, setRegion] = useLocalStorage('region', null)
```

Comment:
are you using local storage instead of context so it stays there when you leave/return to the app? (asking because I should possibly be doing the same for quick exit and I'm not)

Reply:
Yes that's right!